docs/dev-proc/source-code-structure.md

```diff
@@ -65,14 +57,61 @@ Force-pushes to `<platform>/rel_vX.Y.Z`, `<platform>/develop` or
   happen it would be better to have it fixed by original author than the person
   that tries to upstream it some time later.
 
-Force-pushes to  `<platform>/release` branches are unconditionally forbidden.
+Force-pushes to  `dasharo` branches are allowed only for rebasing on new
+coreboot releases, according to the guidelines outlined in the next step.
+
+## Rebasing process
+
+1. Update master/main branch to the recent coreboot upstream tag
+   (and fetch tags as well)
+
+    ```bash
+    git checkout master
+    git fetch upstream
+    git pull upstream `git describe --tags upstream/master --abbrev=0`
```

Comment:
Recently coreboot switched to main

Reply:
fixed in 1ed0a83f08334d132029bbed91dd088fea7ee020